File: 52.c
#include<stdio.h>
 
int main(){
 
    int a, b=0, c=0;
    scanf("%d", &a);
 
    for(int i = 0; i < 3; i++){
        b+=a%10;
        a/=10;
    }
 
    for(int i = 0; i < 3; i++){
        c+=a%10;
        a/=10;
    }
 
    if(b==c){
        printf("YES");
    }
    else{
        printf("NO");
    }
 
    return 0;
}

File: 52.py
k = int(input())
 
b = 0
a = 0
 
for i in range(6):
    if i < 3:
        a += k%10
    else:
        b += k%10
    k //=10
 
if a == b:
    print("YES")
else:
    print("NO")
